vagrant/ansible/roles/client.prep/tasks/main.yml

```diff
@@ -15,49 +15,6 @@
 - debug:
     msg: "{{ ctdb_network_public_interfaces }}"
```

Comment:
Should we remove this debug section too?

Reply:
Probably should but in a different PR I guess. That particular change isn't related to the other fixes which remove the mount points and any changes to this commit message at this stage means waiting for the build fixes to go in.

Reply:
Fine.